jmx-scraper/src/main/java/io/opentelemetry/contrib/jmxscraper/JmxScraper.java

```diff
@@ -104,52 +96,42 @@ private static void loadPropertiesFromPath(Properties props, String path)
   }
 
   JmxScraper(JmxScraperConfig config) throws ConfigurationException {
-    this.config = config;
 
-    try {
-      @SuppressWarnings("unused") // TODO: Temporary
-      JmxClient jmxClient = new JmxClient(config);
-    } catch (MalformedURLException e) {
-      throw new ConfigurationException("Malformed serviceUrl: ", e);
+    String serviceUrl = config.getServiceUrl();
+    if (serviceUrl == null) {
```

Comment:
Do you think these values should be validated here once again? They are already validated in JmxScraperConfigFactory.validateConfig which is called from JmxScraperConfigFactory.createConfig method, which is the only public (from other package) way to instantiate JmxScraperConfig